docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx
+sphinx_rtd_theme
```

Comment:
Seems unneeded now

Reply:
`pip install -r requirements.txt` is standard in Python world, it's a lazy and efficient way to let people on Win/Mac install everything at once and run `sphinx-build -M html docs build/docs` on their OS, should they need it. I can remove it if you really want, but I see the benefit (aside from Github auto-stuff) and not the harm.